src/cloudimagedirectory/filter/filter.py

```diff
@@ -20,6 +28,44 @@ def FilterImageByLatestUpdate(latestDate: pd.Timestamp) -> Callable:
     return lambda data: [
         d
         for d in data
-        if d.content is not None
-        and pd.Timestamp(d.content["date"]).replace(tzinfo=pytz.UTC) > latestDate
+        if d.content is not None and get_utc_datetime(d.content["date"]) > latestDate
     ]
+
+
+def FilterImageByUniqueName() -> Callable:
+    """Filter latest images with unique names."""
+    print(f"filter images by unique names")
```

Comment:
Is this a debug leftover? :)

Reply:
I've removed `f`, because the string doesn't have to be an f-string. Otherwise, all filters have a `print` statement.